GVFS/GVFS.Service/Mac/NotificationHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System;
+
+namespace GVFS.Service.Handlers
+{
+    public class NotificationHandler

```

Comment:
Is this class referenced anywhere right now? If not, should we just wait until we start thinking about `GVFS.Service.UI` to implement it?

Reply:
Agreed, if this file is not getting consumed by anything can/should it be removed from this PR?

Reply:
`RepoRegistry::AutoMountRepos()` uses it for sending notifications after mount. I will be needing it in the next PR (auto-mount on user login) - to log notification messages (until platform native notifications become available.)